pkg/proxy/iptables/proxier.go

```diff
@@ -378,13 +378,27 @@ var iptablesJumpChains = []iptablesJumpChain{
 	{utiliptables.TableNAT, kubePostroutingChain, utiliptables.ChainPostrouting, "kubernetes postrouting rules", nil},
 }
 
-var iptablesCleanupOnlyChains = []iptablesJumpChain{}
+func getExistingChains(ipt utiliptables.Interface, table utiliptables.Table) (chains []utiliptables.Chain, err error) {
+	iptablesData := bytes.NewBuffer(nil)
+	if err = ipt.SaveInto(table, iptablesData); err != nil {
+		klog.Errorf("Failed to execute iptables-save for %s: %v", table, err)
+	} else {
+		existingNATChains := utiliptables.GetChainLines(table, iptablesData.Bytes())
+		for chain := range existingNATChains {
+			chainString := string(chain)
+			if strings.HasPrefix(chainString, "KUBE-SVC-") || strings.HasPrefix(chainString, "KUBE-SEP-") || strings.HasPrefix(chainString, "KUBE-FW-") || strings.HasPrefix(chainString, "KUBE-XLB-") {
```

Comment:
👍 - not deleting `KUBE-SEP-*` chains from iptables proxy was causing all sorts of issues on IPVS clean up.